WowPacketParser/SQL/SQLUtil.cs

```diff
@@ -255,6 +255,19 @@ public static string Compare<T>(IEnumerable<Tuple<T, TimeSpan?>> storeList, RowL
                             val1 =  ((string)val1).Replace("\"\"", "\"");
                         }
 
+                        if (val2 is string)
+                        {
+                            // French and CO
+                            val2 = ((string)val2).Replace("‘‘", "‘");
+                            val2 = ((string)val2).Replace("´´", "´");
+                            // Russia
+                            val2 = ((string)val2).Replace("``", "`");
+                            // Taiwan (they look similar to Russia)
+                            val2 = ((string)val2).Replace("ˋˋ", "ˋ");
+                            // Make it comparable
+                            val2 = ((string)val2).Replace("\"\"", "\"");
+                        }
```

Comment:
This allows to compare locale strings when pulled from DB

Reply:
shouldn't comparing french with french work without any replacements?

Reply:
Well, string from sniff has 1 Apostrophe and string from db has because SQL  two. Then it wants to update it. The update adds the second Apostrophe again.

Reply:
I can also change the building of the SQL Part to avoid Double Apostrophe.
How does the core read from db?

WPP allways has Double Apostrophes :/

Reply:
Gonna Update the Escaping Stringify